packages/shared/src/components/modals/post/BookmarkReminderModal.tsx

```diff
@@ -152,4 +154,13 @@ export const BookmarkReminderModal = (
   );
 };
 
-export default BookmarkReminderModal;
+const ModalComponent: typeof BookmarkReminderModal = ({
+  feedContextData,
+  ...props
+}) => (
+  <ActiveFeedContext.Provider value={feedContextData}>
```

Comment:
since `feedContextData` is optional probably avoid rendering `ActiveFeedContext.Provider` so the value does not become undefined.

Reply:
Good catch!